doc/to.md

```diff
@@ -28,8 +28,7 @@ const outer = new SimpleMap({
 });
 
 const keyObservable = canValue.to(outer, "inner.key");
-// canReflect.getValue(keyObservable) returns keyObservable
 
-canReflect.setValue(keyObservable, "aloha");
+keyObservable.value = "aloha";
```

Comment:
Here’s what this section looks like:
<img width="756" alt="screen shot 2018-07-02 at 7 30 13 pm" src="https://user-images.githubusercontent.com/10070176/42195626-9b599994-7e2e-11e8-9a7e-43c5023ad1de.png">